static-web/css/quiz.css

```diff
@@ -0,0 +1,138 @@
+/*
+The entire card has a solid 1px border that is lightblue.
+The card title has a dotted 1px border that is lightgray.
+The title and product image are centered.
+Notice that the text for the description in the image is justified.
+The text for the product specification details and pricing details is bold.
+The availability element extends the full width of the card, with a dark grey background and yellow text.
+
+***** Personal Challenge *****
+-Responsive design
+
+*/
+
+* {
+  box-sizing: border-box;
+}
+
+/*Base Style for Card Layout*/
+.clearfix:after {
+   content: ".";
+   visibility: hidden;
+   display: block;
+   height: 0;
+   clear: both;
+}
+
+.allCards {
+  margin: 0 auto;
+}
+
+.card {
+	border: solid 1px lightblue;
+	margin: 5%;
+}
+
+.cardTitle {
+	text-align: center;
+	border: dotted 1px lightgray;
+	font-weight: bold;
+	font-size: 2em;
+}
+
+.productImg {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+	max-width: 400px;
+	max-height: 400px;
+}
+
+.description {
+	text-align: justify;
+	margin: auto;
+	width: 90%;
+}
+
+.availability {
+	background: darkgrey;
+	color: yellow;
+	text-align: left;
+	padding-left: 2%;
+}
+
+/*Targeted the element instead of adding class because I needed to target all of them and nothing conflicted*/
+h1, h3 {
+	padding: none;
+	margin: 2% 2%;
+}
+
+li {
+	font-weight: bold;
+	list-style: none;
+}
+
+ul {
+	padding-left: 5%;
+	margin: 2%;
+}
+
+/*Changes to 2 column layout*/
+@media (min-width: 700px) {
+
+
+  .card {
+    display: inline-block;
+    float: left;
+    max-width: 45%;
+    margin: 2.5%;
+  }
+
+  .cardTitle {
+    font-size: 20px;
+  }
+
+  .productImg {
+    max-width: 225px;
+  }
+
+}
+
+/*Stays at 2 column layout with larger images and title font*/
+@media (min-width: 1000px) {
+
+
+	.card {
+    max-width: 45%;
+    margin: 2.5%;
+  }
+
+  .cardTitle {
+    font-size: 30px;
+  }
+
+  .productImg {
+    max-width: 275px;
+  }
+
+}
+
+/*Changes to 3 column layout with smaller images and font*/
+@media (min-width: 1200px){
+
+
+  .card {
+    max-width: 30%;
+    margin: 1.66%;
+  }
+
+  .cardTitle {
+    font-size: 20px;
+  }
+
+  .productImg {
+    max-width: 250px;
+  }
+
+
+}
```

Comment:
Always include a new line at the end of every file. Consider adding ensure_newline_at_eof_on_save to your sublime preferences: http://docs.sublimetext.info/en/latest/reference/settings.html#automatic-behavior